packages/gateway/src/features/user/schema.graphql

```diff
@@ -20,7 +20,7 @@ type UserIdentifier { # -> identifier
 }
 
 input SignatureInput {
-  data: String
+  data: String!
```

Comment:
I modified the schema a little bit to simplify the typing of this. I think it's safe to assume all objects of SignatureInput type has a `data` field or no `attachments: []` list actually contain undefineds / nulls